api/routes/api.php

```diff
@@ -33,7 +35,9 @@
     //    // Routes here
     // });
 
-    // Route::group(['middleware' => ['restrictRole:staff']], function () {
-    //     // Routes here
-    // });
+Route::group(['middleware' => ['restrictRole:staff']], function () {
+    Route::post('/orders/new', function (Request $request) {
+        return Order::create($request->all());
+    });
```

Comment:
1. Should create new controller and transfer the logic there
2. Always add validations
```suggestion
    Route::post('/orders/new', [OrderController::class, 'create']);
```